test/flows/maternity_paternity_pay_leave_flow_test.rb

```diff
@@ -38,4015 +50,4038 @@ class MaternityPaternityPayLeaveFlowTest < ActiveSupport::TestCase
     end
 
     context "next_node" do
-      should "have a next node of employment_status_of_mother " \
+      should "have a next node of due_date " \
                "for any response" do
-        assert_next_node :employment_status_of_mother, for_response: "2016-1-1"
+        assert_next_node :due_date, for_response: "england"
       end
     end
   end
 
-  context "question: employment_status_of_mother" do
-    setup do
-      testing_node :employment_status_of_mother
-      add_responses two_carers: "yes",
-                    due_date: "2016-1-1"
-    end
-
-    should "render the question" do
-      assert_rendered_question
-    end
+  context "for after where_does_the_mother_partner_live question" do
+    setup { add_responses where_does_the_mother_partner_live: "england" }
 
-    context "next_node" do
-      should "have a next node of employment_status_of_partner " \
-               "for any response " \
-               "when there are two carers" do
-        assert_next_node :employment_status_of_partner, for_response: "employee"
+    context "question: due_date" do
+      setup do
+        testing_node :due_date
+        add_responses two_carers: "yes"
+        # add_responses where_does_the_mother_partner_live: "england"
       end
 
-      %w[employee worker].each do |response|
-        should "have a next node of mother_started_working_before_continuity_start_date " \
-                 "for a '#{response}' response " \
-                 "when there is a single carer" do
-          add_responses two_carers: "no"
-          assert_next_node :mother_started_working_before_continuity_start_date, for_response: response
-        end
+      should "render the question" do
+        assert_rendered_question
       end
 
-      %w[self-employed unemployed].each do |response|
-        should "have a next node of mother_worked_at_least_26_weeks " \
-                 "for a '#{response}' response " \
-                 "when there is a single carer" do
-          add_responses two_carers: "no"
-          assert_next_node :mother_worked_at_least_26_weeks, for_response: response
+      context "next_node" do
+        should "have a next node of employment_status_of_mother " \
+                 "for any response" do
+          assert_next_node :employment_status_of_mother, for_response: "2016-1-1"
         end
       end
     end
-  end
-
-  context "question: employment_status_of_partner" do
-    setup do
-      testing_node :employment_status_of_partner
-      add_responses two_carers: "yes",
-                    due_date: "2016-1-1",
-                    employment_status_of_mother: "employee"
-    end
 
-    should "render the question" do
-      assert_rendered_question
-    end
+    context "question: employment_status_of_mother" do
+      setup do
+        testing_node :employment_status_of_mother
+        add_responses two_carers: "yes",
```

Comment:
Since the tests inside this context block don't all use the same response for `two_carers`, (in fact 2/3 require a "no" response), I'd push this `add_response` down into the individual tests (i.e. add it to the one test that doesn't already do it).